### DISCUSSION TITLE
Download POI first?

### DISCUSSION THREAD

**Discussion by DolomitesTom** [devemux86/cruiser#274] [Q&A] [answered]
Don't know if this is an issue, normally the poi file is downloaded first before the country map file, right?

There was just a new Italy file but it didn't download the poi file first. I had to download this separately.

**Comment by devemux86** [comment, marked as answer]
When downloading a new map, then its poi is also automatically downloaded.

As the poi files are smaller than maps, they usually finish downloading earlier.
e.g. Italy: map 1.3G, poi 284M

---

If you mean the updates, they are downloaded manually and separately.
